src/SFA.DAS.EAS.Web/Controllers/EmployerAccountTransactionsController.cs

```diff
@@ -41,10 +51,40 @@ public class EmployerAccountTransactionsController : BaseController
         [Route("balance/downloadtransactions")]
         public ActionResult TransactionsDownload(string hashedAccountId)
         {
-            return View(new OrchestratorResponse<TransactionsDownloadResultViewModel>
+            var viewModel = new TransactionDownloadViewModel
             {
-                Data = new TransactionsDownloadResultViewModel { HashedAccountId = hashedAccountId }
-            });
+                AccountHashedId = hashedAccountId,
+                Message = new GetTransactionsDownloadQuery
+                {
+                    StartDate = new TransactionsDownloadStartDateMonthYearDateTime(),
+                    EndDate = new TransactionsDownloadEndDateMonthYearDateTime()
+                }
+            };
+
+            return View(viewModel);
+        }
+
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        [ValidateModelState]
+        [Route("finance/downloadtransactionsdata")]
+        public async Task<ActionResult> TransactionDownloadByDate(TransactionDownloadViewModel model)
```

Comment:
I recommend overloading your `HttpGet` method here to make use of the `ValidateModelState` attribute's post-redirect-get behavior.